Tests/fixtures/clienthints.yml

```diff
@@ -0,0 +1,79 @@
+---
+-
+  user_agent: Mozilla/5.0 (Linux; Android 8.1.0; ZC554KL) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.87 Mobile Safari/537.36
+  headers:
+    Sec-CH-UA: '" Not A;Brand";v="99", "Chromium";v="98", "Google Chrome";v="98"'
+    Sec-CH-UA-Platform: "Android"
+    Sec-CH-UA-Mobile: "?1"
+    Sec-CH-UA-Full-Version: "98.0.4758.87"
+    Sec-CH-UA-Platform-Version: "8.1.0"
+    Sec-CH-UA-Model: "ZC554KL"
+    Sec-CH-Prefers-Color-Scheme: "light"
+  os:
+    name: Android
+    version: 8.1.0
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 98.0.4758.87
+    engine: Blink
+    engine_version: ""
+  device:
+    type: smartphone
+    brand: Asus
+    model: ZenFone 4 Max
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; arm_64; Android 8.1.0; ZC554KL) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.174 YaApp_Android/22.14.1 YaSearchBrowser/22.14.1 BroPP/1.0 SA/3 Mobile Safari/537.36
+  headers:
+    Sec-CH-UA: '" Not A;Brand";v="99", "Chromium";v="96", "Yandex";v="22"'
+    Sec-CH-UA-Platform: "Android"
+    Sec-CH-UA-Mobile: "?1"
+    Sec-CH-UA-Full-Version: "22.1.4.706"
```

Comment:
Is there a known reason, why theses versions do not match?

Reply:
this is not a typo, I double-checked
![image](https://user-images.githubusercontent.com/1337066/155879231-1635b82a-218e-42ac-8ee6-9ffd2dc02525.png)

Reply:
only today the answer came, from yandex


> Hello!
> Sorry for the delay in responding.
> The versions do not match, because in the User Agent we write the version of the application, and in the client hits a more accurate version of the assembly.